### ISSUE TITLE
Request: RotaryCraft support!

### ISSUE THREAD

**Issue by BionicFelps** [Electrical-Age/ElectricalAge#215]
RotaryCraft seems to have a lot of points for mod interaction with this mod.  It has a two-factor power system (torque, speed), it has a concept of heat as well, it also has an API for easy integration.  I'd recommend allowing this mod's heat items to operate on RotaryCraft heat (heat sinks, heat furnace, etc.) and also creating a rotational power -> ELN converter (generator).  Power can already go the other direction via the ELN->Other mod block and an RF powered RotaryCraft generator.

I'm currently making a Modpack that will include both mods and see a lot of potential here.  The pack will NOT include Thermal Expansion, BuildCraft, or any of the other "traditional" power mods, as it's designed to highlight a number of less-known mods, and I believe I found a nice second ecosystem of mods that should surprise people!

RotaryCraft also has an addon called "ReactorCraft" that adds complex nuclear reactors, which take a lot of resources and know-how to construct but produce vast amounts of rotational power.

**Comment by BionicFelps** [comment]
I'm assuming that you would have made other to ELN blocks with the ELN to other blocks if you wanted RF to be used for power, assumption being that RF is not two-factor power, so you would just have to automatically make it at 50v, 200v, and 800v and a variable amps depending... Also energy cells would trivialize the use of batteries.  

RotaryCraft has no easy power storage itself, along with the two-factor power system, so it seems it would take care of those concerns automatically!  In fact, ELN batteries would be a nice way to store rotarycraft power at that point...

**Comment by nanakisan** [comment]
Hmmmm ELN's power system is about as equally complex as RotaryCrafts. However ultimately this would require Dolu having Reikas permission. Reika is very and I mean very strict on compatibility of mods with his. If anything Reika might add the compatibility himself instead of allowing Dolu to add a block for it.

**Comment by BionicFelps** [comment]
This is true, he's also quite strict on modpack devs, I didn't want to bring it to Reika himself because he actually makes an electrical mod with full RotaryCraft support.  I mistakenly thought that his API was open for mod interaction though.  (As essentially the writer of the API is still in control of what's possible, I didn't check..)

**Comment by ghost** [comment]
I'm aware this is a bit late, but Reika has mentioned that he's not against adding converters to his mods, as long as he can talk to Dolu first.  ReikaKalseki/Reika_Mods_Issues#109

Personally, once that hurdle's been passed, I'd like to be able to add this to the Survival Industry modpack.
